deploy/non-olm/v1.2.0/operator.yml

```diff
@@ -143,6 +143,13 @@ rules:
   - '*'
   verbs:
   - '*'
+- apiGroups:
+  - config.openshift.io
```

Comment:
This is also in the role and not hte new clusterrole where it belongs. All this would do is give access to these resources in this namespace, which won't get us what we want.

https://github.com/konveyor/mig-operator/blob/master/deploy/non-olm/latest/operator.yml#L149